### PR TITLE
MISUV-525: Update flow to utilise new income sources page

### PR DESCRIPTION
**Pull request by scullythornton1**

### CODE REVIEW

File: app/controllers/individual/business/BusinessAccountingMethodController.scala [hunk 1]
```diff
@@ -22,8 +22,7 @@ import forms.individual.business.AccountingMethodForm
 import javax.inject.{Inject, Singleton}
 import models.Yes
 import models.common.AccountingMethodModel
-import models.individual.incomesource.RentUkPropertyModel
-import models.individual.subscription.Business
+import models.individual.incomesource.{IncomeSourceModel, RentUkPropertyModel}
```
Comment (JSimmons072019) [inline app/controllers/individual/business/BusinessAccountingMethodController.scala:25]: Clean up imports

Comment (scullythornton1) [reply app/controllers/individual/business/BusinessAccountingMethodController.scala]: updated

File: app/controllers/individual/business/PropertyAccountingMethodController.scala [hunk 2]
```diff
@@ -21,7 +21,7 @@ import config.AppConfig
 import forms.individual.business.AccountingMethodPropertyForm
 import javax.inject.{Inject, Singleton}
 import models.common.AccountingMethodPropertyModel
-import models.individual.incomesource.{AreYouSelfEmployedModel, RentUkPropertyModel}
+import models.individual.incomesource.{AreYouSelfEmployedModel, IncomeSourceModel, RentUkPropertyModel}
```
Comment (JSimmons072019) [inline app/controllers/individual/business/PropertyAccountingMethodController.scala:24]: clean up imports

File: app/utilities/CacheUtil.scala [hunk 3]
```diff
@@ -20,7 +20,7 @@ import config.AppConfig
 import models.common.{AccountingMethodModel, AccountingMethodPropertyModel, AccountingYearModel, BusinessNameModel}
 import models.individual.business.address.Address
 import models.individual.business.{AccountingPeriodModel, BusinessPhoneNumberModel, BusinessStartDateModel, MatchTaxYearModel}
-import models.individual.incomesource.{AreYouSelfEmployedModel, RentUkPropertyModel}
+import models.individual.incomesource.{AreYouSelfEmployedModel, IncomeSourceModel, RentUkPropertyModel}
```
Comment (JSimmons072019) [inline app/utilities/CacheUtil.scala:23]: imports

File: it/controllers/individual/business/BusinessAccountingMethodControllerISpec.scala [hunk 4]
```diff
@@ -21,7 +21,7 @@ import helpers.IntegrationTestConstants._
 import helpers.IntegrationTestModels.{keystoreData, testMatchTaxYearYes}
 import helpers.servicemocks.{AuthStub, KeystoreStub}
 import models.common.AccountingMethodModel
-import models.individual.incomesource.RentUkPropertyModel
+import models.individual.incomesource.{IncomeSourceModel, RentUkPropertyModel}
```
Comment (JSimmons072019) [inline it/controllers/individual/business/BusinessAccountingMethodControllerISpec.scala:24]: imports

File: test/controllers/individual/business/BusinessAccountingMethodControllerSpec.scala [hunk 5]
```diff
@@ -20,13 +20,11 @@ import controllers.ControllerBaseSpec
 import forms.individual.business.AccountingMethodForm
 import models.common.AccountingMethodModel
 import models.individual.incomesource.RentUkPropertyModel
```
Comment (JSimmons072019) [inline test/controllers/individual/business/BusinessAccountingMethodControllerSpec.scala:22]: import

File: test/utilities/TestModels.scala [hunk 6]
```diff
@@ -21,7 +21,7 @@ import java.time.LocalDate
 import utilities.individual.TestConstants.{testFirstName, testLastName, testNino, testUtr}
 import models.individual.business.address.{Address, Country, ReturnedAddress}
 import models.individual.business._
-import models.individual.incomesource.{AreYouSelfEmployedModel, RentUkPropertyModel}
```
Comment (JSimmons072019) [inline test/utilities/TestModels.scala:24]: imports

File: test/views/individual/incometax/subscription/CheckYourAnswersViewSpec.scala [hunk 7]
```diff
@@ -23,11 +23,11 @@ import models.DateModel
 import models.common.{AccountingMethodModel, AccountingMethodPropertyModel, AccountingYearModel, BusinessNameModel}
 import models.individual.business._
 import models.individual.business.address.Address
-import models.individual.incomesource.{AreYouSelfEmployedModel, RentUkPropertyModel}
+import models.individual.incomesource.{AreYouSelfEmployedModel, IncomeSourceModel, RentUkPropertyModel}
```
Comment (JSimmons072019) [inline test/views/individual/incometax/subscription/CheckYourAnswersViewSpec.scala:26]: import

File: app/controllers/individual/business/PropertyAccountingMethodController.scala (hunk 2 above)
Comment (scullythornton1) [reply app/controllers/individual/business/PropertyAccountingMethodController.scala:24]: done

File: app/utilities/CacheUtil.scala (hunk 3 above)
Comment (scullythornton1) [reply app/utilities/CacheUtil.scala:23]: there is no need for import update.

File: it/controllers/individual/business/BusinessAccountingMethodControllerISpec.scala (hunk 4 above)
Comment (scullythornton1) [reply it/controllers/individual/business/BusinessAccountingMethodControllerISpec.scala:24]: done

File: test/controllers/individual/business/BusinessAccountingMethodControllerSpec.scala (hunk 5 above)
Comment (scullythornton1) [reply test/controllers/individual/business/BusinessAccountingMethodControllerSpec.scala:22]: no need

File: test/utilities/TestModels.scala (hunk 6 above)
Comment (scullythornton1) [reply test/utilities/TestModels.scala:24]: done

File: test/views/individual/incometax/subscription/CheckYourAnswersViewSpec.scala (hunk 7 above)
Comment (scullythornton1) [reply test/views/individual/incometax/subscription/CheckYourAnswersViewSpec.scala:26]: done

**Pull request merged**